vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,220 @@
+ 
+The PAM implementation relies on a form definition attached to the process definition. Entando transforms the PAM representation into a JSON schema form that can be rendered to the end user.
+ 
+### Summary Card
+ 
+The Summary Card MFE provides a view into aggregate data for the process
 implementation. The rendered information includes a total value, a trend
-value, and a selector for timeframe. The summary card allows the
-application developer to select a request for rendering the requested
-information which maps to a call in the underlying engine to provide the
-summarized data.
-
-### PAM Implementation
-
-The PAM implementation of the summary card widget relies on the PAM
-custom query functionality. The PAM PDA engine exposes configuration
-that allows users/developers to define a custom query in a config file
-that will be executed to render the summary card. This implementation
-allows the data rendered on the summary cards to be changed via
-configuration. In the application there is a properties file that allows
-the implementer to provide a custom query for each of the cards.
-
-## Totals Over Time
-
-The totals over time MFE provides a dual axis line/bar graph providing
-trend information about the process environment. The MFE also provides
-three summary values for comparison over the same time period.
-
-### PAM Implementation
-
-The PAM implementation of the totals over time MFE utilizes custom
-queries to fetch the summary data rendered in the chart. The queries
-used in the implementation are defined in configuration files in the MFE
-and can be updated to render data specific to a given implementation.
-
-# Customizing the Process Driven Application
-
+value, and a timeframe selector. The Summary Card allows the
+application developer to select a request for rendering information. This request maps to a call in the underlying engine and provides the summarized data.
+ 
+The PAM implementation of the Summary Card widget relies on the PAM custom query functionality. The PAM PDA engine exposes a configuration file where the custom query can be defined. This allows user customization of the data rendered on the summary cards. The application contains a "properties" file where the implementer can submit a custom query for each of the cards.
+ 
+### Totals Over Time
+ 
+The Totals Over Time MFE provides a dual axis line/bar graph displaying
+trend information about the process environment. Three summary values can be compared over a single time period.
+ 
+The PAM implementation of the Totals Over Time MFE utilizes custom queries to fetch the summary data rendered in the chart. The queries used in the implementation are defined in configuration files in the MFE and can be updated to render implementation specific data.
+ 
+## Process Automation Manager (PAM)
+ 
+Process Automation Manager (PAM) is a business process automation engine built and maintained by Red Hat.
+ 
+## Business Process Modeling Notation (BPMN)
```

Comment:
Agreed, if BPMN and DMN aren't referenced later, we can remove them. I think BFF is still worth keeping because the microservice in this implementation is the definition of a backend for a frontend, acting as a facade layer to another service.

Reply:
will do